graspy/embed/omni.py

```diff
@@ -204,24 +201,34 @@ def fit(self, graphs, y=None):
         # Embed
         self._reduce_dim(omni_matrix)
 
+        # Reshape to tensor
+        self.latent_left_ = self.latent_left_.reshape(
+            self.n_graphs_, self.n_vertices_, -1
+        )
+        if self.latent_right_ is not None:
+            self.latent_right_ = self.latent_right_.reshape(
+                self.n_graphs_, self.n_vertices_, -1
+            )
+
         return self
 
     def fit_transform(self, graphs, y=None):
         """
         Fit the model with graphs and apply the embedding on graphs. 
-        n_dimension is either automatically determined or based on user input.
+        n_components is either automatically determined or based on user input.
 
         Parameters
         ----------
-        graphs : list of graphs
+        graphs : list of graphs, or array-like
```

Comment:
I feel like this part is a bit confusing, maybe something like 
"list of Nx.graph or ndarray, or ndarray:
if list of ndarray, must be (n x n)
if single ndarray, must be ngraph x n x n "

Reply:
Or in the below just clarify "if single array-like...."

Also, would this really work with a list of lists of lists? Just curious whether it's truly array like or just array